content/_index.de.md

```diff
@@ -54,64 +61,68 @@ Organic Maps ist frei von Trackern und anderen schädlichen Dingen:
 - Kein lästiger E-Mail-Spam
 - Keine Push-Benachrichtigungen
 - Keine Crapware
-- ~~Keine Pestizide~~ - Rein und organisch!
+- ~~Keine Pestizide~~ Rein und organisch!
```

Comment:
the translation of 'pure and organic'  in content/_index.de.md has been changed to 'rein und nachhaltig' (pure and sustainable), which sounds better to me. It should be changed here as well.